docs/reference/cluster/state.asciidoc

```diff
@@ -1,6 +1,18 @@
 [[cluster-state]]
 === Cluster State
 
+[[cluster-state-api-request]]
+==== {api-request-title}
+
+`GET /_cluster/state` +
+
+`GET /_cluster/state/{metrics}` +
+
+`GET /_cluster/state/{metrics}/{indices}`
```

Comment:
I would just include `GET /_cluster/state/{metrics}/{indices}`

It will be clear from the parameter definitions that these are optional.